primefaces/src/main/java/org/primefaces/config/PrimeConfiguration.java

```diff
@@ -55,6 +55,7 @@ public class PrimeConfiguration {
     private final boolean earlyPostParamEvaluation;
     private final boolean moveScriptsToBottom;
     private boolean csp;
+    private boolean policyProvided;
     private String cspPolicy;
```

Comment:
Please update `docs\12_0_0\gettingstarted\configuration.md` and `\docs\12_0_0\core\contentsecuritypolicy.md` to update with all this new info

Reply:
I've added the documentation as requested